misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -36,10 +40,17 @@ class AwsSqsJobQueueModule(
 
     install(ServiceModule(keyOf<RepeatedTaskQueue>(ForSqsConsumer::class)))
 
-    install(ExecutorServiceModule.withFixedThreadPool(
+    // We use an unbounded thread pool (not really as it's is bounded by a dynamic flag) for
```

Comment:
I don't believe this one is bounded by the dynamic flag. isn't this the number of background tasks? so it would be 1 per queue?

Reply:
oh this is for the handlers. this is super confusing. we should rename this.

consumer: background task. 1 per queue per vm
receivers: N per queue, configured via a dynamic flag, load balanced across the cluster
handlers: unbounded, although in reality bounded by the number of receivers.

Reply:
yep. I added because https://github.com/cashapp/misk/pull/1346#discussion_r363553606. I think maybe you meant in the `ForSqsReceiving` `ExecutorService`?